src/editTasks.js

```diff
@@ -0,0 +1,8 @@
+export default function editTasks(li) {
+  const taskSpan = li.querySelector('.list__item-description');
+  taskSpan.addEventListener('click', () => {
+    const newContent = prompt('Edit task');
+    if (newContent === null || newContent === '') return;
+    taskSpan.textContent = newContent;
+  });
+}
```

Comment:
- [ ] Please make sure when the user edits a task, the change is then reflected in the local storage.